x-pack/plugins/searchprofiler/public/application/components/profile_tree/profile_tree.tsx

```diff
@@ -30,6 +31,7 @@ export const ProfileTree = memo(({ data, target, onHighlight }: Props) => {
   } catch (e) {
     // eslint-disable-next-line no-console
     console.error(e);
```

Comment:
Now that we surface the error in the UI, do we still need this?

Reply:
Will remove!